Rocket.Chat/ChatControllerVideoconference.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ChatControllerVideoconference.swift
+//  RocketChat
+//
+//  Created by Luís Machado on 18/12/2017.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+
+extension ChatViewController {
+
+    func getVideoconferenceURL() -> String? {
+        let settings = AuthManager.isAuthenticated()?.settings ?? AuthSettings()
+        guard let roomId = subscription?.rid, let serverId = settings.serverId, let prefix = settings.videoChatPrefix else { return nil }
```

Comment:
What do you think about breaing this guard in more lines? We're doing that in other files.